### PR TITLE
Fetch Funtion for API Request

### PR DESCRIPTION
**Pull request by Simplemart17**
- Setup fetch API request

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 
 export default function App() {
+  const URL = 'https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest';
+  useEffect(() => {
+    fetch(URL, {
+      headers: {
+        'X-CMC_PRO_API_KEY': '07cff7a2-6b5c-4328-86d1-77fceed0f357'
```
Comment (uchemukolo) [inline App.js:9]: You cannot expose the API key like that
Create a `.env` and add it to the `.gitignore` file. In the root directory and add the api key like this:
`API_KEY=07cff7a2-6b5c-4328-86d1-77fceed0f357`
then do `process.env.API_KEY` where you want to use it

Comment (Simplemart17) [reply App.js:9]: Alright thanks for the feedback

File: App.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (rebirthtobi) [inline App.js:1]: can you put this in a file of its own and put it inside `lib` folder.
The full path will be `src/lib/request.js`

File: App.js [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
 import { StyleSheet, Text, View } from 'react-native';
 
 export default function App() {
+  const URL = 'https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest';
```
Comment (rebirthtobi) [inline App.js:5]: there is no need attaching it to a react component, just make it a function that can be called by any component at any time

File: .env [hunk 4]
```diff
@@ -0,0 +1 @@
+CRYPTO_API_KEY=07cff7a2-6b5c-4328-86d1-77fceed0f357
```
Comment (rebirthtobi) [inline .env:1]: can you remove this file as it is not needed anymore

**Pull request merged**